kafka/source/pkg/reconciler/resources/labels.go

```diff
@@ -14,13 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apis
+package resources
 
-import (
-	eventingv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
+const (
+	// controllerAgentName is the string used by this controller to identify
+	// itself when creating events.
+	controllerAgentName = "kafka-source-controller"
 )
 
-func init() {
-	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, eventingv1alpha1.SchemeBuilder.AddToScheme)
+func getLabels(name string) map[string]string {
```

Comment:
I think the others use `Labels` (see eventing, apiServerSource), you could than use that in the `kafkasource_test`, see others